content/MBC1.md

```diff
@@ -26,8 +26,8 @@ higher (01/21/41/61).
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
-External RAM is often battery buffered, allowing to store game positions
-or high score tables, even if the Game Boy is turned off, or if the
+External RAM is often battery buffered to store game positions
```

Comment:
Removing the comma and merging the clauses makes them unclear. "...battery-buffered, which allows it to store..." would be better.